src/main/java/com/github/skjolberg/packing/Dimension.java

```diff
@@ -188,8 +188,8 @@ public long getVolume() {
 		return volume;
 	}
 	
-	public boolean isEmpty() {
-		return width <= 0 || depth <= 0 || depth <= 0;
+	public boolean nonEmpty() {
+		return width > 0 && depth > 0 && height > 0;
```

Comment:
Here's the bug: the last thing compared was `depth <= 0` instead of `height <= 0`